getting-started/kubernetes/trying-ebpf.md

```diff
@@ -0,0 +1,190 @@
+   ```bash
+   kubectl get endpoints kubernetes
+   ```
+   The output should look like this:
+   ```bash
+   kubectl get endpoints kubernetes
+   NAME         ENDPOINTS           AGE
+   kubernetes   <IP>:<port>         43h
+   ```
+   Record the `<IP>` and `<port>`.
+1. Modify the `kubernetes_service_host` and `kubernetes_service_port` variables in the config map at the top of the manifest.  Set `kubernetes_service_host` to the IP you recorded above.  Set `kubernetes_service_port` to the port.
+   ```yaml
+   kind: ConfigMap
+   apiVersion: v1
+   metadata:
+     name: calico-config
+     namespace: kube-system
+   data:
+     ...
+     kubernetes_service_host: "<IP>"
+     kubernetes_service_port: "<port>"
+     ...
+   ```
+   
+1. Disable `kube-proxy` on your cluster.  The following command adds a node selector to the `kube-proxy` daemon set that won't match anything:
+
+   ```bash
+   kubectl patch ds -n kube-system kube-proxy -p '{"spec":{"template":{"spec":{"nodeSelector":{"non-calico": "true"}}}}}'
+   ```  
+   
+   Since the selector won't match anything, `kube-proxy` won't run anywhere but it can be easily re-enabled by removing the selector.
+   
+1. Apply the {{site.prodname}} manifest using the following command.
+
+   ```bash
+   kubectl apply -f calico-bpf.yaml
+   ```
+   
+1. Verify that the calico-node pods start and become ready and that the `kube-proxy` pods shut down:
+   ```bash
+   watch kubectl get po -n kube-system
+   ```
+   Should give output like this:
+   ```
+   NAME                                            READY   STATUS    RESTARTS   AGE
+   calico-kube-controllers-78d64db6fd-dlqwm        1/1     Running   0          28m
+   calico-node-6ffzz                               1/1     Running   0          13m
+   calico-node-9ccpx                               1/1     Running   0          13m
+   calico-node-mx2v4                               1/1     Running   0          13m
+   calico-node-n9pc9                               1/1     Running   0          13m
+   coredns-5644d7b6d9-bpddg                        1/1     Running   0          46m
+   coredns-5644d7b6d9-glfr7                        1/1     Running   0          46m
+   etcd-host-name                                  1/1     Running   0          46m
+   kube-apiserver-host-name                        1/1     Running   0          46m
+   kube-controller-manager-host-name               1/1     Running   0          46m
+   kube-scheduler-host-name                        1/1     Running   0          46m
+   ``` 
+
+### Next steps
+
+**Tools**
+
+- [Install and configure calicoctl]({{site.baseurl}}/getting-started/calicoctl/install)
+
+**Networking**
+
+- If you are using the default BGP networking with full-mesh node-to-node peering with no encapsulation, go to [Configure BGP peering]({{site.baseurl}}/networking/bgp) to get traffic flowing between pods.
+
+**Security**
+
+- [Secure Calico component communications]({{site.baseurl}}/security/comms/crypto-auth)
```

Comment:
Given it's just tech preview, I suggest deleting the secure Calico components link.  That feels more like the kind of thing you would do if heading towards production rather than playing with a tech preview.